src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/component/ActionPlanValidator.kt

```diff
@@ -15,7 +15,7 @@ class ActionPlanValidator {
       if (it <= 0) {
         errors.add(FieldError(field = "numberOfSessions", error = Code.CANNOT_BE_NEGATIVE_OR_ZERO))
       }
-    } ?: errors.add(FieldError(field = "numberOfSessions", error = Code.CANNOT_BE_EMPTY))
```

Comment:
This can only be checked at the point of submitting the action plan when it is fully entered as apposed to validation that can be done on incremental changes.